File: notification-service/src/test/java/com/eirsteir/coffeewithme/notification/web/api/NotificationControllerTest.java
package com.eirsteir.coffeewithme.notification.web.api;

import com.eirsteir.coffeewithme.notification.config.ModelMapperConfig;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.extension.ExtendWith;
import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
import org.springframework.context.annotation.Import;
import org.springframework.test.context.junit.jupiter.SpringExtension;

@Import({ModelMapperConfig.class})
@WebMvcTest(NotificationController.class)
@ExtendWith(SpringExtension.class)
class NotificationControllerTest {

  @BeforeEach
  void setUp() {}
}

File: notification-service/src/test/kotlin/com/eirsteir/coffeewithme/notification/web/api/NotificationControllerTest.kt
package com.eirsteir.coffeewithme.notification.web.api

import org.junit.jupiter.api.BeforeEach
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.web.servlet.MockMvc

@SpringBootTest
@AutoConfigureMockMvc
internal class NotificationControllerTest {

    @Autowired
    private lateinit var mockMvc: MockMvc

    @BeforeEach
    fun setUp() {
    }
}